test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -925,7 +926,9 @@ def revert_mom(self, mom, force=False):
         name = mom.shortname
         if mom.platform == 'cray' or mom.platform == 'craysim':
             try:
-                self.server.manager(MGR_CMD_DELETE, NODE, None, '')
+                if self.del_all_nodes:
```

Comment:
If I understand this code then you are trying to delete and recreate node once a test suite (aka class level). is that what you are doing here? if yes then can you please put a comment saying that.